prime-router/metadata/fhir_transforms/senders/default-sender-transform.yml

```diff
@@ -28,4 +28,16 @@ elements:
     condition: 'Bundle.entry.resource.ofType(MessageHeader).meta.tag.code.empty()'
     resource: 'Bundle.entry.resource.ofType(MessageHeader)'
     bundleProperty: '%resource.meta.tag.code'
-    value: [ '"P"' ]
\ No newline at end of file
+    value: [ '"P"' ]
+
+  - name: provider-namespace-id
+    condition: '%resource.value.empty().not()'
+    resource: 'Bundle.entry.resource.ofType(Practitioner).identifier[0]'
+    bundleProperty: '%resource.extension("https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority-namespace-id").value[x]'
+    value: [ '"NPI"' ]
+
+  - name: provider-universal-id
+    condition: '%resource.value.empty().not()'
+    resource: 'Bundle.entry.resource.ofType(Practitioner).identifier[0]'
+    bundleProperty: '%resource.extension("https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority-universal-id").value[x]'
+    value: [ '"urn:oid:2.16.840.1.113883.4.6"' ]
```

Comment:
Where did this value come from?

Reply:
I got it from the covid-19.schema. There's a function there that checks if we have a valid NPI and appends that if it is valid.
` mapper: ifNPI(ordering_provider_id, NPI^2.16.840.1.113883.4.6^ISO)`